037-WindowsVirtualDesktop/Student/10-Monitor-Manage-Performance-Health.md

```diff
@@ -0,0 +1,47 @@
+
+**NOTE:** Scaling Automation was deployed in Challenge 9
+
+#### WVD Monitoring
+   - Azure offers numerous ways to monitor resources your solution should offer a view that targets the WVD service diagnostics and host performance and events
+   - The service needs to have the ability to report on both live information and historical information for at least 30 days
+   - Information needs to be presented in an easy to consume dashboard and offer a way to filter views based on which region you're attempting to monitor
+
+#### Cost Optimization and Session Host scaling
+   - Assume each region starts seeing staff logon between 8am and 9:30am, there is no sudden spike in logons. The office is closed by 7pm.  
+   - Users can be forcibly logged off after receiving a 5 minute warning if they are still connected to an active host
+	- Approximately 5% of users in each region may attempt to connect from home each night and WVD should be available to them off-peak across each region and meets the cost saving targets set by the company
+   - WVD Session Hosts should be gradually powered on as morning peak demand increases and existing hosts should be filled up before additional VMs are powered on to reduce excess capacity.
+
+## Success Criteria
+
+To be successful for this challenge you should be able to demonstrate the following;
+
+- Identify the top errors preventing users from connecting over the past 24 hours
+- Select a host pool and show a view with the following information for each WVD host: New Session Status, Stack Version, Current Sessions, Available Sessions
+- Display recent user activity in WVD activity for a specific user including their most recent login and history within the past 30 days
+- Identify performance metrics of a WVD host which has a currently connected user and demonstrate how you can view the "User Input Delay per Process" object and "Max Input Delay" counters for that VM.
+- Enable automatic scaling of WVD session hosts to start and stop machines during peak/off-peak hours across each region. Consider how the peak/off-peak hours and load balancing will influence the configuration. Assume for the purpose of this workshop that VM sizing and user density is designed for a 1CPU per Session to help with testing in a limited size environment.
+
+**Tips**
+
+> _The logging information may take a few minutes to appear in the monitoring portal. You'll also need to generate user traffic by connecting to WVD after enabling the diagnostics for information to flow through._
+
+> _To simulate connection errors you may wish to attempt logins with invalid credentials, closing the client during the logon attempt and deliberately using a different account between the logon to the WVD portal and the WVD session host which should fail._
+
+> _Simulate an unavailable session host by manually stopping the WVD service "RdAgent"_
```

Comment:
Please don't use angle brackets on this text. We don't want it to appear as a quote on the rendered page.